### DISCUSSION TITLE
supabase getSession!

### DISCUSSION THREAD

**Discussion by ix2md** [supabase/supabase#22413] [Questions]
Done ✅

**Comment by Kylar13** [comment]
Having the same issue as well, and this used to work fine

<img width="749" alt="image" src="https://github.com/supabase/supabase/assets/7138244/2b759c37-c389-403c-b79a-b7204ef8d42d">

**Reply by Kylar13** [reply]
Tried checking versions of packages, updating, reverting to versions we had when it worked, and nothing seems to fix it
